airflow/plugins_manager.py

```diff
@@ -205,6 +219,10 @@ def make_module(name, objects):
     if p.stat_name_handler:
         stat_name_handlers.append(p.stat_name_handler)
     global_operator_extra_links.extend(p.global_operator_extra_links)
+    # Only register Operator links if its ``operators`` property is not None
+    # So that we can only attach this links to a specific Operator
+    operator_extra_links.extend([
+        ope for ope in p.operator_extra_links if ope.operators is not None])
```

Comment:
`if ope.operators` if we change it to not be Optional

Reply:
Updated